packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -143,11 +154,30 @@ export class BlobManager {
             values.map((entry) => detached ? this.detachedBlobIds.add(entry) : this.blobIds.add(entry));
         }
         this.logger.sendTelemetryEvent({ eventName: "AttachmentBlobsLoaded", count });
+        if (redirectTable) {
+            this.redirectTable = new Map(redirectTable);
+            this.logger.sendTelemetryEvent({ eventName: "BlobRedirectTableLoaded", count: redirectTable.length });
```

Comment:
I'd appreciate adding this property to existing event above and not create new events.